include/fmt/os.h

```diff
@@ -394,7 +394,7 @@ struct ostream_params {
 
 FMT_END_DETAIL_NAMESPACE
 
-constexpr detail::buffer_size buffer_size;
+constexpr detail::buffer_size buffer_size{};
```

Comment:
Could you add a short comment clarifying that this is a workaround for clang and which version.

Reply:
Sure. Since Apple has their own numbering for llvm and clang, I've just commented on the Xcode version where this is known to occur.